stl/inc/flat_set

```diff
@@ -271,13 +270,10 @@ public:
     void insert_range(_Rng&& _Range) {
         const size_type _Old_size = size();
 
-        _Container& _Cont = _Get_cont();
-        if constexpr (requires { _Cont.append_range(_STD forward<_Rng>(_Range)); }) {
-            _Cont.append_range(_STD forward<_Rng>(_Range));
+        if constexpr (requires { _Mycont.append_range(_STD forward<_Rng>(_Range)); }) {
+            _Mycont.append_range(_STD forward<_Rng>(_Range));
         } else {
-            for (const auto& _Val : _Range) {
-                _Cont.insert(_Cont.end(), _Val);
-            }
+            _Mycont.insert_range(_Mycont.end(), _STD forward<_Rng>(_Range));
         }
         _Restore_invariants_after_insert<false>(_Old_size);
     }

```

Comment:
```
    void insert(initializer_list<_Kty> _Ilist) {
        _Insert_range<false>(_Ilist.begin(), _Ilist.end());
    }

    void insert(_Tsorted, initializer_list<_Kty> _Ilist) {
        _Insert_range<true>(_Ilist.begin(), _Ilist.end());
    }
```
These methods can use `cont.insert(iter, ilist)` for insertion; temporarily kept unchanged for tidiness.